client/src/components/vote/vote-single/vote-single.html

```diff
@@ -0,0 +1,155 @@
+      <h3 class="md-title">Description</h3>
+      <md-list flex>
+        <md-list-item class="md-2-line">
+          <md-icon>
+            cloud_upload
+            <md-tooltip>File upload date</md-tooltip>
+          </md-icon>
+          <div class="md-list-item-text">
+            <h3>
+              {{ $ctrl.rating.current.entry.upload_date | date:'d MMM yyyy' }}
+            </h3>
+            <p>
+              {{ $ctrl.rating.current.entry.upload_date | date:'EEEE' }},
+              {{ $ctrl.rating.current.entry.upload_date | date:'H:mm' }}
+            </p>
+          </div>
+        </md-list-item>
+        <md-list-item class="md-2-line">
+          <md-icon>
+            image
+            <md-tooltip>File resolution</md-tooltip>
+          </md-icon>
+          <div class="md-list-item-text">
+            <h3>
+              {{ $ctrl.rating.current.entry.resolution / 1000000 | number:2 }} Mpix
+            </h3>
+            <p>
+              {{ $ctrl.rating.current.entry.width }} ×
+              {{ $ctrl.rating.current.entry.height }}
+            </p>
+          </div>
+        </md-list-item>
+        <md-list-item class="md-2-line"
+            ng-show="$ctrl.rating.current.history">
+          <md-icon>
+            history
+            <md-tooltip>File history</md-tooltip>
+          </md-icon>
+          <div class="md-list-item-text">
+            <h3>
+              {{ $ctrl.rating.current.history.length }}
+              version<span ng-if="$ctrl.rating.current.history.length > 1">s</span>
+            </h3>
+            <p>
+              last one at
+              {{ $ctrl.rating.current.history[0].timestamp | date:'d MMM yyyy' }}
+            </p>
+          </div>
+          <md-button class="md-secondary md-icon-button"
+              ng-href="{{ '//commons.wikimedia.org/wiki/File:' + $ctrl.rating.current.entry.name + '#filehistory' }}"
+              target="_blank">
+            <md-icon>link</md-icon>
+          </md-button>
+        </md-list-item>
+      </md-list>
+    </div>
+  </div>
+</div>
+
+<img class="invisible" ng-src="{{ $ctrl.getImageName($ctrl.rating.next) }}">
+<div class="central-error" layout="column" layout-align="center center"
+   ng-show="$ctrl.error">
+  <h2 class="md-display-1">{{$ctrl.error.message}}</h2>
+  <h2 class="md-headline">{{$ctrl.error.detail}}</h2>
+</div>
+<div class="central-error" layout="column" layout-align="center center"
+   ng-show="$ctrl.round.status !== 'active' && !$ctrl.error">
+  <h2 class="md-headline">this round is {{$ctrl.round.status}}</h2>
+</div>
+<div class="central-error" layout="column" layout-align="center center"
+   ng-show="$ctrl.images && !$ctrl.images.length">
+  <h2 class="md-headline">you voted on all images in the round</h2>
+  <span>good job! :)</span>
+  <br />
+  <span><md-icon>new_releases</md-icon> You can edit your votes on <a href="" ui-sref="main.juror.vote-edit({id: $ctrl.round.id})">this page</a></span>
```

Comment:
Getting:

`angular.js:14110 Error: Could not resolve 'main.juror.vote-edit' from state 'main.vote'`